.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1", "8.2"]
+                php: ["8.1", "8.2", "8.3"]
```

Comment:
As we do not bump up the PHP version in composer, I would be in favour of still testing 8.0. Preferably the lowest (8.0) and highest (8.3) versions if we want to reduce the number of builds in matrix and add single builds for 8.1 and 8.2 in `include`.